docs/content/dagster-cloud/account/setting-up-alerts.mdx

```diff
@@ -44,10 +48,14 @@ Currently, Slack and email notifications are supported.
 
 4. Click **+ Create alert policy**.
 
+5. From the `Alert Policy Type` drop-down, choose `Run Alert` or `Schedule/Sensor Alert`.
+
 5. In the **Create alert policy** window, fill in the following:
 
    - **Alert policy name** - Enter a name for the alert policy. For example, `slack_urgent_failure`
    - **Description** - Enter a description for the alert policy
+
+  For run-based alerts, fill out these additional options:
```

Comment:
This section was renamed to run-based alert policies - why is this sentence needed here?